go.mod

```
@@ -1,19 +1,16 @@
 module github.com/bombsimon/logrusr/v2
 
-go 1.13
+go 1.17
```

`logr` itself is supported back to [1.16](https://github.com/go-logr/logr/blob/master/go.mod) and since golang is still supporting the [last two releases](https://endoflife.date/go) I think we can keep this at 1.16.

```suggestion
go 1.16
```

---

done